src/main/java/com/alkemy/ong/infrastructure/database/mapper/UserEntityMapper.java

```diff
@@ -56,7 +60,8 @@ public UserEntity toEntity(User user) {
         .lastName(user.getLastName())
         .photo(user.getPhoto())
         .password(user.getPassword())
+        .role(roleEntityMapper.toEntity(user.getRole()))
```

Comment:
@JackyOS please check if this works fine with the Comment feature you're working on

Reply:
It works perfect. Thanks. 👍